external/fv3fit/fv3fit/keras/_models/recurrent.py

```diff
@@ -0,0 +1,611 @@
+from typing import Dict, Any, Iterable, Hashable, Optional, Sequence, Tuple, List
+import tensorflow as tf
+import xarray as xr
+import os
+from ..._shared import Predictor, ArrayPacker, io
+from ._filesystem import get_dir, put_dir
+from .normalizer import LayerStandardScaler
+import yaml
+import numpy as np
+
+
+class BPTTModel(Predictor):
```

Comment:
I should describe the purpose of this object in the docstring as well (it's only for training, not prediction).

Reply:
I'll make BPTTModel a private class, and pull some code from `train.py` in here to use as a function instead of having this class be public. The parent class will also be removed.